README.md

```diff
@@ -374,6 +374,29 @@ You are able to use environment variables to customize identity params in event
 
 You can use [serverless-dotenv-plugin](https://github.com/colynb/serverless-dotenv-plugin) to load environment variables from your `.env` file.
 
+## Per-function Disabling
+
+Disabling a function that you don't want to be handled by `serverless-offline` can be done by adding `serverless-offline-ignore: true` to the function's spec within
+`serverless.yml`. For example:
+```
+functions:
+  ignoredfunction:
+    handler: handler.ignoredfunction
+    events:
+      - http:
+          method: get
+          path: /ignored-function
+    serverless-offline-ignore: true
```

Comment:
Adding a new property for a function will require you to also extend the validation schema for function - that's why you're getting the warning and forcing validation to be disabled is definitely not the way to go. Alternatively, maybe it would be better to keep all `serverless-offline`-related config in corresponding custom section and add a property e.g. `ingnoredFunctions` or something like that?

Reply:
Added [function-property validation](https://github.com/dherault/serverless-offline/pull/1245/commits/795b2c33a74e811482179849ad4ac3bc87cb8e18#diff-015eb49a9424f9d5210a4012878eb50b70f17e9dba36ebab21974701e5e8e1c5R55) so that the `serverless-offline-ignore` can be inline.